tests/php/ImgConverterTest.php

```
@@ -1038,4 +1038,28 @@ public function test_queue_unconverted_library_images_skips_already_converted():
+				. str_repeat( "\x00", 16 ) . "\xFF\xD9"
+		);
+
+		$converter = $this->make_converter( array( 'conversionFormat' => 'webp' ) );
+		$result    = $converter->convert_image( $file, 'webp' );
+
+		$this->assertFalse( $result );
+
+		$full_rel = str_replace( wp_normalize_path( ABSPATH ), '', wp_normalize_path( $file ) );
+		$info     = Img_Converter::get_img_info();
+		$this->assertContains( $full_rel, $info['skipped']['webp'] ?? array() );
+	}
```

PHPMD (2.15.0)

[error] The method test_convert_image_skips_gain_map_sources is not named in camelCase. (undefined)

(CamelCaseMethodName)